frontend/src/views/CellCards/components/CellCard/components/common/Table/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import {
+  StyledTable,
+  StyledHead,
+  StyledHeadCell,
+  StyledRow,
+  StyledCell,
+  TableWrapper,
+} from "./style";
+
+interface TableProps<T> {
+  columns: Array<Extract<keyof T, string>>;
+  rows: T[];
+  columnIdToName?: Record<Extract<keyof T, string>, string>;
```

Comment:
This is an optional property so users can set the column names to complex strings that cannot be encoded as properties in the row objects (for example "% cells" cannot be a property name).

If unset, then the property names in each `row` are capitalized and set as column names.